tee-worker/omni-executor/Makefile

```diff
@@ -51,7 +51,7 @@ endif
 
 .PHONY: start-gramine-server
 start-gramine-server: all
-	$(GRAMINE) tee-bridge
+	$(GRAMINE) omni-executor ${ARGS}
```

Comment:
There is renaming above "omni-executor" -> "executor-worker". Here it still uses "omni-executor"?